monai/apps/pathology/handlers/prob_map_producer.py

```diff
@@ -25,6 +25,10 @@
     Engine, _ = optional_import("ignite.engine", IgniteInfo.OPT_IMPORT_VERSION, min_version, "Engine")
 
 
+@deprecated(
+    since="0.8",
+    msg_suffix="use `monai.handler.ProbMapProducer` (with `monai.data.wsi_dataset.SlidingPatchWSIDataset`) instead.",
```

Comment:
I don't quite understand why it's replaced by `monai.data.wsi_dataset.SlidingPatchWSIDataset`?

Thanks.

Reply:
`ProbMapProducer` uses meta information that `SlidingPatchPatchReader` provides to create the probability maps. There has been some changes in those meta information when refactoring pathology datasets.

`monai.apps.handler.ProbMapProducer` <-> `monai.apps.data.MasedInferencePatchWSIDataset`
`monai.handler.ProbMapProducer` <-> `monai.data.wsi_dataset.SlidingPatchWSIDataset`

Reply:
@Nic-Ma is this clear?
Also I have added the keys to enum so they will be reused in any related dataset.